src/mozilla_sec_eia/cli.py

```diff
@@ -9,6 +9,8 @@
 import logging
```

Comment:
The CLI isn't working for me (I'm getting "command not found" when I try to run `validate_archive` for example). Do we need a `setup.py` in the repo or is an entry point defined in some other way?

Reply:
Yeah I just disabled the check. Not sure why it was coming up now when I haven't seen it in other repos, but shouldn't be a problem now